com.unity.visualeffectgraph/CHANGELOG.md

```diff
@@ -65,6 +65,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Fix unexpected Spawn context execution ordering
 - Fix incorrect buffer type for strips
 - Enabled an optimization for motion vectors, storing projected positions for vertices instead of the transform matrix
+- Move Sanitize from PreProcess to PostProcess to avoid loss of data due to model migration [Case 1344645](https://issuetracker.unity3d.com/product/unity/issues/guid/1344645/)
```

Comment:
I don't think users care about "sanitize was moved from pre process to post process", the log needs to be more user oriented (i.e what does it fix)

Reply:
Thanks, I update the log at this revision : https://github.com/Unity-Technologies/Graphics/pull/4971/commits/f5a6be58f9efba2b28dfb6203d9e904fb847e262